src/main/kotlin/no/nav/familie/klage/oppgave/OppgaveService.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.klage.oppgave
+
+import no.nav.familie.klage.behandling.BehandlingService
+import no.nav.familie.klage.fagsak.FagsakService
+import no.nav.familie.klage.infrastruktur.sikkerhet.SikkerhetContext
+import no.nav.familie.klage.oppgave.OppgaveUtil.lagFristForOppgave
+import no.nav.familie.kontrakter.felles.oppgave.IdentGruppe
+import no.nav.familie.kontrakter.felles.oppgave.OppgaveIdentV2
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.kontrakter.felles.oppgave.OpprettOppgaveRequest
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Service
+class OppgaveService(
+    private val oppgaveClient: OppgaveClient,
+    private val fagsakService: FagsakService,
+    private val behandlingService: BehandlingService
+) {
+
+    fun opprettBehandleSakOppgave(behandlingId: UUID) {
+        val fagsak = fagsakService.hentFagsakForBehandling(behandlingId)
+        val behandling = behandlingService.hentBehandling(behandlingId)
+
+        val oppgaveRequest = OpprettOppgaveRequest(
+            ident = OppgaveIdentV2(ident = fagsak.hentAktivIdent(), gruppe = IdentGruppe.FOLKEREGISTERIDENT),
+            saksId = fagsak.eksternId, // fagsakId fra fagsystem
+            tema = fagsak.stønadstype.tilTema(), //
+            oppgavetype = Oppgavetype.VurderKonsekvensForYtelse,
+            fristFerdigstillelse = lagFristForOppgave(LocalDateTime.now()),
+            beskrivelse = "Klagebehandling i ny løsning",
+            enhetsnummer = behandling.behandlendeEnhet,
+            behandlingstype = "ae0058",
```

Comment:
`ae0058` er en magic string her og burde legges inn in `Behandlingstema` i kontrakter, så kan man bruke den både for å sende inn og søk på oppgaver.

Reply:
fixd [ae803fe](https://github.com/navikt/familie-klage/pull/78/commits/ae803fec0a92cbc48cc24ddc44cf2f45b038aae4)